runtime/ajuna/src/xcm_config.rs

```diff
@@ -390,28 +390,31 @@ pub type LocalOriginToLocation = SignedToAccountId32<RuntimeOrigin, AccountId, R
 
 impl pallet_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
+	type Currency = Balances;
+	type CurrencyMatcher = ();
 	type SendXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, ()>; // Prohibit sending arbitrary XCMs from users of this chain
 	type XcmRouter = XcmRouter;
-	type ExecuteXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>; // Allow any local origin in XCM execution.
-	type XcmExecuteFilter = Nothing; // Disable generic XCM execution. This does not affect Teleport or Reserve Transfer.
+	type ExecuteXcmOrigin = EnsureXcmOrigin<RuntimeOrigin, LocalOriginToLocation>;
+	// Allow any local origin in XCM execution.
+	type XcmExecuteFilter = Nothing;
+	// Disable generic XCM execution. This does not affect Teleport or Reserve Transfer.
```

Comment:
The formatting put the comments on the wrong line, they describe the item above now. Can you move them on top of the item they describe?

Reply:
Adjusted formatting!